src/lib_ccx/ccx_encoders_helpers.c

```diff
@@ -179,6 +179,8 @@ unsigned encode_line(struct encoder_ctx *ctx, unsigned char *buffer, unsigned ch
 			bytes += 2;
 			buffer += 2;
 			break;
+		case CCX_ENC_ASCII: // TODO: encode ASCII text
```

Comment:
Not sure if this is TODO or just not needed at all.

Reply:
Then why is there CCX_ENC_ASCII in enum?